src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/44.diff.sql

```diff
@@ -0,0 +1,12 @@
+SET XACT_ABORT ON
+BEGIN TRANSACTION
+
+DROP PROCEDURE IF EXISTS dbo.GetInstanceBatchesByTimeStamp
+GO
+
+COMMIT TRANSACTION
+
+DROP INDEX IF EXISTS IX_Instance_Watermark_Status_CreatedDate ON dbo.Instance
```

Comment:
This was only used for the sproc?

Reply:
yeah, It was created as part of this work